Lib/idlelib/debugger.py

```diff
@@ -10,12 +10,20 @@
 
 
 class Idb(bdb.Bdb):
+    """Idle debugger, based on the bdb debugger."""
 
     def __init__(self, gui):
         self.gui = gui  # An instance of Debugger or proxy of remote.
-        bdb.Bdb.__init__(self)
+        self.botframe = None
+        super(Idb, self).__init__()
 
     def user_line(self, frame):
+        """
```

Comment:
I never start docstrings with a blank line, but I checked
https://www.python.org/dev/peps/pep-0257/ and this is allowed for a multiline string.
IDLE calltips currently will not see the following summary line, so I made a note to fix it.